fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/TabItemTokens.kt

```diff
@@ -171,4 +173,9 @@ open class TabItemTokens : IControlToken, Parcelable {
     open fun textTypography(tabItemInfo: TabItemInfo): TextStyle {
         return FluentTheme.aliasTokens.typography[FluentAliasTokens.TypographyTokens.Caption2]
     }
+
+    @Composable
+    open fun indicatorColor(tabItemInfo: TabItemInfo): StateBrush? {
```

Comment:
return default fluent values here. In this case the values are same as the text color

Reply:
If the indicatorColor returns null then the indicator color will be set to textColor, the logic for that has been added when setting the indicator color